e2e-tests/playwright/tests/functional/channels/notifications/notification.spec.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect, test} from '@e2e-support/test_fixture';
+
+test('MM-T483 Channel-wide mentions with uppercase letters', async ({pw, pages, browser, headless, browserName}) => {
+    test.skip(
+        headless && browserName !== 'firefox',
+        'Works across browsers and devices, except in headless mode, where stubbing the Notification API is supported only in Firefox and WebKit.',
+    );
+
+    // Initialize setup and get the required users and team
+    const {team, adminUser, user} = await pw.initSetup();
+
+    // Log in as the admin in one browser session and navigate to the "town-square" channel
+    const {page: adminPage} = await pw.testBrowser.login(adminUser);
+    const adminChannelPage = new pages.ChannelsPage(adminPage);
+    await adminChannelPage.goto(team.name, 'town-square');
+    await adminChannelPage.toBeVisible();
+
+    // Stub the Notification in the admin's browser to capture notifications
+    await pw.stubNotification(adminPage, 'granted');
+
+    // Log in as the regular user in a separate browser and navigate to the "off-topic" channel
+    const otherBrowser = new pw.TestBrowser(browser);
```

Comment:
And its usage here

Reply:
This has been updated and we'll stick in using `await pw.testBrowser.login(user)` when needing separate browser session.

Reply:
yes thats much understandable and clear as well